solarforecastarbiter/reports/templates/html/pending.html

```diff
@@ -5,6 +5,7 @@ <h1 id="report-title">{{ report.report_parameters.name }}</h1>
 {% endblock %}
 <div class="alert alert-info">
   Report generation is pending. We will try and fetch the report again in 30 seconds, or you can try reloading this page.
+  If no report is available after 30 minutes, try recomputing the report.
```

Comment:
do you have benchmark to suggest that 30 minutes is the right number? in my experience it works in less 5 minutes or not at all.

Reply:
Nope. I just picked a number that I thought would definitely be long enough considering possibly more congestion than there is now